### PR TITLE
Add initial docsite

### PR DESCRIPTION
**Pull request by ianks**
Figured this gem needed a little love on the website, so I took some time to write out some docs. Let me know what you think! @solnic 

Feel free to edit as you please btw

❤️

### CODE REVIEW

File: docsite/source/third-party-integrations.html.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+### GraphQL
+
+You can easily instrument GraphQL applications with `dry-monitor` with a simple custom tracer.
+
+```ruby
+require 'dry-monitor'
+
+$notifications = Dry::Monitor::Notifications.new(:my_app)
+$notifications.register_event(:graphql)
```
Comment (solnic) [inline docsite/source/third-party-integrations.html.md:28]: Any chance we could avoid using global vars?

Comment (ianks) [reply docsite/source/third-party-integrations.html.md:28]: Yes I just used it for illustration purposes. IRL you would prob register this on a container, but I feel like that would make the example way more verbose. Thoughts?

Also, I did a thing 😊 https://github.com/rmosolgo/graphql-ruby/pull/3776

Comment (solnic) [reply docsite/source/third-party-integrations.html.md:28]: @ianks OK but can we at least use a constant for that and add a comment saying that IRL you'd use DI with dry-system or whatever you prefer?

Comment (ianks) [reply docsite/source/third-party-integrations.html.md:28]: yup!

**Comment by ianks** [comment]
@solnic updated the docs with the [new feature i added to graphql-ruby](https://github.com/rmosolgo/graphql-ruby/pull/3776)

**Pull request merged**